pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.6</version>
+        <relativePath/>
+    </parent>
+    <groupId>com.datmt</groupId>
+    <artifactId>keycloak-springboot-microservice</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>keycloak-springboot-authentication</name>
+    <description>Spring boot keycloak authen</description>
+
+    <properties>
+        <java.version>11</java.version>
+        <keycloak.version>24.0.1</keycloak.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.auth0</groupId>
+            <artifactId>java-jwt</artifactId> <!-- JWT Dependency needed to decode and obtain roles from the Keycloak Token -->
```

Comment:
Let's move the comment to the above of each dependency/plugin tag as follows:

<!-- comment -->
<dependency>
...